engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/DiagnosticStorage.scala

```diff
@@ -61,3 +61,7 @@ final class DiagnosticStorage(initDiagnostics: Seq[Diagnostic] = Seq())
     new DiagnosticStorage(this.diagnostics)
   }
 }
+
+object DiagnosticStorage {
+  def empty(): DiagnosticStorage = new DiagnosticStorage()
```

Comment:
`create()` or `createEmpty()` might be better to indicate that a new instance of this mutable class is always created.

Reply:
Renamed in b3ce70420d775f494a45f93d8588df784da5aaa1